### PR TITLE
Transaction Submitter

### PR DESCRIPTION
**Pull request by Sidu28**
Adds transaction submitter for proofs.  Requires the setup of a JSON file with file paths, and other values

### CODE REVIEW

File: tx_submitoor/main.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	)
+
+	// Handling commands based on the 'command' flag
+	switch *command {
+	case "WithdrawalFieldsProof":
+		calldata, err := submitter.SubmitVerifyAndProcessWithdrawalsTx(*withdrawalProofConfigFile, *submitTransaction)
+		if err != nil {
+			log.Panic().Msgf("failed to submit withdrawal proof: %s", err)
+		}
+		log.Info().Msgf("Withdrawal proof submitted with calldata: %s", calldata)
```
Comment (dan-u410) [inline tx_submitoor/main.go:60]: might be nice to add a case for the VerifyWithdrawalCredentials proof generation:

```
case: "verifyCredentialsProof":
      calldata, err := submitter.SubmitVerifyWithdrawalCredentialsTx(*withdrawalProofConfigFile, *submitTransaction)
```

Comment (Sidu28) [reply tx_submitoor/main.go:60]: I missed pushing my final commit which included that change, just did.  Thanks for the call out

File: tx_submitoor/main.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+type Config struct {
+	RPC          string `json:"RPC_URL"`
+	PrivateKey   string `json:"PRIVATE_KEY"`
+	ChainID      uint64 `json:"CHAIN_ID,required"`
+	CacheExpire  int    `json:"CACHE_EXPIRE" envDefault:"1000"`
+	BeaconAPIURL string `json:"BEACON_API_URL"`
```
Comment (dan-u410) [inline tx_submitoor/main.go:19]: Hi, with how your parseConfig() function is written, I believe you want to use the struct tag env and not json here

File: tx_submitoor/main.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+	cfg := parseConfig()
+
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+
+	// Defining flags for all the parameters
+	command := flag.String("command", "", "The command to execute")
+	withdrawalProofConfigFile := flag.String("withdrawalProofConfig", "", "Withdrawal proof config file")
+	submitTransaction := flag.Bool("submitTransaction", false, "Submit transaction to the chain")
+
```
Comment (dan-u410) [inline tx_submitoor/main.go:32]:
```
// Parse the flags
flag.Parse()
```

These flags are not being set in the code

**Pull request merged**